src/ScalarAuthClient.js

```diff
@@ -76,6 +76,35 @@ class ScalarAuthClient {
         return defer.promise;
     }
 
+    getScalarPageTitle(url) {
+        const defer = Promise.defer();
+
+        let scalarPageLookupUrl = SdkConfig.get().integrations_rest_url + '/api/widgets/title_lookup';
```

Comment:
Why not use the URL preview endpoint from the media repo?

Reply:
ftr this ends up being https://scalar-staging.riot.im/scalar/api/api/widgets/title_lookup (double api)

Reply:
Ahh... thanks for pointing that out @turt2live. I'm not really familiar with the media repo, though it looks like you're right, that would have been a simpler solution. It looks like this can be switched over easily at any point though.

I'll have a look in to the URL string duplication now (it's working fine in dev)...

Reply:
The duplication is because the string in the `config.json` has the `api` route defined in the REST endpoint.

Examples:
https://riot.im/develop/config.json
https://riot.im/app/config.json

Reply:
Yep, you're absolutely right. Thanks for catching that one! (slightly screwy config. on my end to blame for that one, I think). Fixed now.